libfreerdp/gdi/gdi.c

```diff
@@ -1081,6 +1092,13 @@ static BOOL gdi_surface_bits(rdpContext* context, const SURFACE_BITS_COMMAND* cm
 				         cmd->bmp.bitmapDataLength, size);
 				goto out;
 			}
+
+            // Bound check, check that the paste area is valid,
+            // and determine if the picture is in the paste area.
+            const RECTANGLE_16 rect = { (UINT16)cmd->destLeft, (UINT16)cmd->destTop,
```

Comment:
something missing?

Reply:
>  if (!is_rect_valid(&rect, gdi->width, gdi->height)) {
                WLog_ERR(TAG, "cmd->bmp is outside the scope of the gdi area.");
                goto out;
            }
I need to add this code to judge the rect area.